tests/functional/base_pages/question.page.js

```diff
@@ -35,6 +35,7 @@ class QuestionPage extends BasePage {
 
   switchLanguage(language_code) { return 'a[href="?language_code=' + language_code + '"]'; }
 
+  hubLink(route) { return 'a[href="/' + route + '/"]'; }
```

Comment:
As `route` will always be `questionnaire` there is no need to make it a parameter.

Reply:
Fixed. Thanks!